src/main/scala/com/metl/model/BackendAdaptor.scala

```diff
@@ -455,6 +455,7 @@ class ManagedCache[A <: Object,B <: Object](name:String,creationFunc:A=>B,cacheC
     .memoryStoreEvictionPolicy(cacheConfig.memoryEvictionPolicy)
     .persistence(new PersistenceConfiguration().strategy(PersistenceConfiguration.Strategy.NONE))
     .logging(false)
+  cacheConfig.timeToLiveSeconds.foreach(s => cacheConfiguration.timeToLiveSeconds(s))
```

Comment:
okay, so if it's returning this, then it's probably okay in both possible usages.